tests/ci_tests/test_coffee_maker_mains.py

```diff
@@ -0,0 +1,45 @@
+"""Tests ensuring each coffee_maker script with a __main__ guard imports cleanly."""
+
+from __future__ import annotations
+
```

Comment:
Added docstring for the function 'clean_string'.
```suggestion
def clean_string(input_string: str) -> str:
```